consensus/common_test.go

```diff
@@ -266,17 +285,26 @@ func decideProposal(
 		panic("Failed to createProposalBlock. Did you forget to add commit for previous block?")
 	}
 
+	var (
```

Comment:
In line 276 the original PR (L285 in that case) is calling the newly created CreateProposalBlockAndBlob function. You are here creating a proposal w/o a blob.